packages/manager/modules/support/src/modal/translations/Messages_fr_FR.json

```
@@ -0,0 +1,6 @@
+{
+  "modal_beta_help_center_description": "Nouvelle interface de vos demandes d'assistance",
+  "modal_beta_help_center_please_note": "Découvrez la dès maintenant en basculant sur la nouvelle interface et faites-nous part de votre avis.",
```

i am not stotaly sure but may be it's like this :
```suggestion
  "modal_beta_help_center_please_note": "Découvrez là dès maintenant en basculant sur la nouvelle interface et faites-nous part de votre avis.",
```